mailet/icalendar/src/main/java/org/apache/james/transport/mailets/ICalendarParser.java

```diff
@@ -134,10 +136,28 @@ public String getMailetInfo() {
             ByteArrayInputStream inputStream = new ByteArrayInputStream(icsContent);
             return Stream.of(Pair.of(key, builder.build(inputStream)));
         } catch (IOException e) {
-            LOGGER.error("Error while reading input: " + icsContent, e);
+            if (LOGGER.isErrorEnabled()) {
+                String icsContentAsString;
```

Comment:
Extract "icsContentAsString" in an other method to make this more readable, please ;-)

Reply:
Done